doc/source/parameters/dem/floating_walls.rst

```diff
@@ -0,0 +1,37 @@
+Floating Walls
+-------------------
+.. note::
+    Floating wall is a temporary (its start and end times are defined) flat wall, generally used for holding the particles during the filling and before the discharge stage.
```

Comment:
This does not need to be a note, it's the explanation of what it is.

Reply:
corrected.